src/components/SlateEditor/plugins/toolbar/ToolbarButton.tsx

```diff
@@ -107,18 +110,40 @@ const StyledButton = styled(ButtonV2)`
   }
 `;
 
+const getTitle = (
+  i18n: CustomI18n,
+  t: TFunction,
+  type?: string,
+  noTitle?: boolean,
+  disabled?: boolean,
+): string | undefined => {
+  if (noTitle) return;
+
+  if (disabled) {
+    const disabledTranslation = `editorToolbar.disabled.${type}`;
+    const translationExists = i18n.exists(disabledTranslation);
+
+    if (translationExists) return t(disabledTranslation);
+  }
+
+  return t(`editorToolbar.${type}`, options);
+};
```

Comment:
Trenger vi egentlig dette?

Reply:
Dette er for å kunne håndtere en mer beskrivende tooltip for disablede elementer i toolbar, hvis ikke disablet oversettelse er lagt inn vil oppførselen være som før. Hva tenkte du på?

Reply:
Får høre med NDLA!